### PR TITLE
Feature/demo

### PR DESCRIPTION
**Pull request by nlewxxs**
Add different personality types. 

By default, we create: 

- 1 `CheatShortTerm`
- 1 `CheatLongTerm`
- Remaining are all `Honest`

### CODE REVIEW

**Comment by kushk2** [comment]
uh oh

File: s
Comment (dhruvds12) [inline s:1]: Why is there a file named "s"

**Review by dhruvds12 — APPROVED** [review]

File: s
Comment (nlewxxs) [reply s:1]: oops !

**Review by kishan-v — APPROVED** [review]
Looks good to me: can tidy up some style things down the line if we have time

**Review by johanjino — COMMENTED** [review]
Approved

**Pull request merged**